setup.py

```diff
@@ -32,17 +32,17 @@
     classifiers=[
         'License :: OSI Approved :: MIT License',
         'Framework :: Django',
-        'Framework :: Django :: 1.8',
-        'Framework :: Django :: 1.9',
-        'Framework :: Django :: 1.10',
+        'Framework :: Django :: 1.11',
+        'Framework :: Django :: 2.0',
+        'Framework :: Django :: 2.1',
```

Comment:
https://github.com/pypa/warehouse/issues/4090

Reply:
@AGASS007 great!
are you planning to release a new version soon?